app/src/main/shared/actions/_AuthActions.ts

```diff
@@ -0,0 +1,63 @@
+import { Action, Dispatch } from "redux";
+import { AxiosResponse, AxiosError } from "axios";
+
+import { Authenticated, AuthenticationError, Unauthenticated, TypeKeys } from "../actionTypes/AuthTypes";
+import { decodeToken, Token, isExpired, parseModellingGroups } from "../modules/JwtToken";
+import { setShinyToken } from "../sources/LoginSource";
+
+
```

Comment:
Could we make the three actions here static members of a class called `AuthActions`? Then we could refer to them as `AuthActions.loadToken`, `AuthActions.authenticated` etc. which I think would be great for disambiguation and readability.